### PR TITLE
Add instructions for Rosetta to SETUP.md

### PR DESCRIPTION
**Pull request by ebeastlake**
Updates SETUP.MD to encourage Rosetta usage and provide installation instructions.

## Links

Jira ticket: [https://codedotorg.atlassian.net/browse/SL-282](SL-282)

## Testing story

Will run changes by @cat5inthecradle.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -77,7 +77,16 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
...
+These steps may need to change over time as 3rd party tools update to have versions compatible with the new architecture.
+
+0. _(M1 Mac users only)_ Install Rosetta 2.
+
+  - Check if Rosetta is already installed: `/usr/bin/pgrep -q oahd && echo Yes || echo No`
+  - If not, install Rosetta using
+    - `softwareupdate --install-rosetta` (launches the Rosetta installer) or
+    - `/usr/sbin/softwareupdate --install-rosetta --agree-to-license` (skips installer and license agreement)
 
 1. Open your Terminal. These steps assume you are using **zsh**, the default shell for OSX.
```
Comment (cat5inthecradle) [inline SETUP.md:98]: Is there a step needed to ensure you open your terminal with rosetta?

Comment (cat5inthecradle) [reply SETUP.md:98]: For example, I have Rosetta installed, but my terminal is ARM.

image

Comment (fisher-alice) [reply SETUP.md:98]: Here's a [reference](https://www.bigbinary.com/books/learn-rubyonrails-book/setting-up-mac) to enable Rosetta. Does this work for you, @cat5inthecradle ?

File: SETUP.md [hunk 2]
```diff
@@ -105,7 +114,7 @@ These steps are for OSX devices, including Apple Macbooks running on [Apple Sili
 1. Install **rbenv** via `brew install rbenv`
 
 1. Install **Ruby**
-    1. For non-M1 systems, running `rbenv install` from the project root directory should be sufficient
+    1. For non-M1 systems (including M1 systems using Rosetta), running `rbenv install` from the project root directory should be sufficient.
```
Comment (fisher-alice) [inline SETUP.md:124]: I didn't know that M1 systems using Rosetta didn't need the following special config. Thanks for clarifying this.

**Review by fisher-alice — APPROVED** [review]
Thanks for updating this, Emily! Other than adding a step to ensure terminal opens with Rosetta, LGTM!

**Pull request merged**